custom/enikshay/tasks.py

```diff
@@ -48,8 +48,11 @@ def enikshay_task():
     # `toggles.UATBC_ADHERENCE_TASK` enabled
     domains = toggles.UATBC_ADHERENCE_TASK.get_enabled_domains()
     for domain in domains:
-        updater = EpisodeUpdater(domain)
-        updater.run()
+        try:
```

Comment:
Thanks for adding this. Ideally, try/except blocks must be present for a task run on each case, so that if a case fails the task proceeds on other cases, so at least when I wrote this, I added those blocks per case-run. Apart from adding this domain-level, if you came across any uncuaght exceptions at case-level they should be included in a try/except as well.,